packages_rs/nextclade/src/tree/tree.rs

```diff
@@ -35,8 +35,20 @@ impl TreeNodeAttr {
 }
 
 #[derive(Clone, Serialize, Deserialize, Validate, Debug)]
-pub struct TreeNodeAttrInt {
+pub struct TreeNodeAttrF64 {
   pub value: f64,
+
+  #[serde(flatten)]
+  pub other: serde_json::Value,
+}
+
+impl TreeNodeAttrF64 {
+  pub fn new(value: f64) -> Self {
+    Self {
+      value,
+      other: serde_json::Value::default(),
+    }
+  }
 }
```

Comment:
The `TreeNodeAttrF64` type to be used for `placement_prior`. Just like `TreeNodeAttr`, but with f64 inside.

Reply:
Ah - got it, the Int was how you had suggested to use the priors before.

Does this F64 mean I can't wrap the value in quotes in JSON, or does this not matter?

Would both of these work for deserialization?
```
value: "3.123"
value: 3.123
```

Reply:
The default should probably be NAN, so set it explicitly as I'm not sure default isn't 0.

Reply:
Nope, the "3.123" is string. If the Rust type is `f64`, serde will expect the number (without quotation marks).

Reply:
The default is `None`, and can be handled as such (with `Option::unwrap*()` or `Option::map*()` ). No need to invent magic/bogus values.

Reply:
> it would be nice if we could deal with either situation

I don't see why it would be nice. It would be super ugly in my view. Why would you want to complicate things and do 2 useless manual conversions:

 - number to string in your Python code
 - string to number in Rust (and handle potential failures)

when a type system can handle that.

Number is a number. Not string. If it's a number, it's guaranteed to be there and can be used without errors (the parsing and type checking is handled by serde-json).

Just remove the useless conversion code in Python and everything will be working again (and if it requires rerunning the pipeline, for the time being just convert them manually with sed or something)